packages/node_modules/samples/browser-plugin-meetings/app.js

```diff
@@ -46,17 +46,14 @@ function connect() {
     });
   }
 
-  if (!webex.internal.device.registered) {
-    webex.internal.device
-      .register()
+  // Register our device with Webex cloud
+  if (!webex.meetings.registered) {
```

Comment:
would it make sense to move this conditional `webex.meetings.registered` inside of `webex.meetings.register()`?
so like inside `register()` if `webex.meetings.registered` just reject or something? what happens if the user accidentally tries to register twice?

Reply:
Good call. I will follow the pattern of the other plugins and log that it is already registered and immediately resolve the promise.